src/mount/mount.ceph.c

```diff
@@ -268,6 +268,21 @@ static int parse_options(const char *data, struct ceph_mount_info *cmi)
 			/* ignore */
 		} else if (strcmp(data, "nofail") == 0) {
 			/* ignore */
+		} else if (strcmp(data, "fs") == 0) {
+			if (!value || !*value) {
+				fprintf(stderr, "mount option fs requires a value.\n");
+				return -EINVAL;
+			}
+			data = "mds_namespace";
+			skip = false;
+		} else if (strcmp(data, "mds_namespace") == 0) {
+			fprintf(stdout, "warning: mount option mds_namespace is deprecated, please use 'fs=<fs_name>'.\n");
```

Comment:
Let's skip this warning. It'll just be annoying and we will always accept `mds_namespace`.

Reply:
Sure，done.
Thanks.